src/compute-types/src/dataflows.rs

```diff
@@ -64,6 +65,10 @@ pub struct DataflowDescription<P, S: 'static = (), T = mz_repr::Timestamp> {
     /// only data is suppressed. (This is consistent with how frontier advancements can also
     /// happen before the `as_of`.)
     pub until: Antichain<T>,
+    /// The initial as_of when the collection is first created. Filled only for materialized views.
```

Comment:
Can you elaborate the comment please? I don't understand whether this is expected to stay the same forever per global ID, or whether it can advance during restarts/new replicas. Also explain how it differs from `as_of`.

Reply:
Stays the same forever. I've extended the comment with this, and also renamed this to `initial_storage_as_of`.